docs/api-guide/schemas.md

```diff
@@ -52,6 +52,12 @@ If your schema is static, you can use the `generateschema` management command:
 ./manage.py generateschema --file openapi-schema.yml
 ```
 
+You can also pass `tag_objects` as an option to `generateschema` command.
+
+```shell script
+$ python manage.py generateschema --tag_objects='[{"name": "pet", "description": "Pets operations"}, {"name": "store", "description": "Store operations"}]'
+```
```

Comment:
I'm skeptical that the CLI is the right place for this (rather than a Python file with a SchemaGenerator subclass in it.)

Reply:
Thanks for your suggestion. I am more inclined towards custom subclass for CLI.
It can be difficult to pass JSON as an argument in CLI.  

Should I remove it? Even I am not convinced about this.

Reply:
I'm not sure we should add this at all now... 

> Just override `get_schema()` to add the top level tags key.

I'd think that would be sufficient. 

Folk might want more in their own project but we need to keep base implementation as simple as we can.

Reply:
I do agree with you. 
Let me create a GitHub gist. We can add it's link to the documentation which will help folks. Also, the core library will be cleaner.

Reply:
Super thanks @dhaval-mehta 🙏

Reply:
Hi @dhaval-mehta We already have an example here. https://www.django-rest-framework.org/api-guide/schemas/#get_schemaself-request

Happy to look at edits but I'm not sure we need more — folks who want this will know where to add it.